routes/dao_coin_exchange_with_fees.go

```diff
@@ -494,41 +496,46 @@ func (fes *APIServer) GetQuoteCurrencyPriceInUsdEndpoint(ww http.ResponseWriter,
 		return
 	}
 
-	usdPrice, err := fes.GetQuoteCurrencyPriceInUsd(requestData.QuoteCurrencyPublicKeyBase58Check)
+	usdPrice, bestBid, bestAsk, err := fes.GetQuoteCurrencyPriceInUsd(requestData.QuoteCurrencyPublicKeyBase58Check)
 	if err != nil {
 		_AddBadRequestError(ww, fmt.Sprintf("GetQuoteCurrencyPriceInUsd: Problem getting quote currency price in USD: %v", err))
 		return
 	}
 
-	res := GetQuoteCurrencyPriceInUsdResponse{UsdPrice: usdPrice}
+	res := GetQuoteCurrencyPriceInUsdResponse{
```

Comment:
While you're here can you rename UsdPrice to MidPrice? It will break my AMM but it's more clear and I'll fix it it's fine.